server/app/views/dev/DatabaseSeedView.java

```diff
@@ -39,11 +39,16 @@ public DatabaseSeedView(BaseHtmlLayout layout, ObjectMapper objectMapper) {
     this.objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
   }
 
-  public Content render(
+  /**
+   * Renders a page for a developer to view seeded data. This is only available in non-prod
+   * environments.
+   */
+  public Content SeedDataView(
```

Comment:
**suggestion (blocking):** methods in Java must be camelCase beginning with a lowercase letter, it's worthwhile to read through the Java styleguide: https://google.github.io/styleguide/javaguide.html
```suggestion
  public Content seedDataView(
```

Reply:
updated. thanks!